admin/app/views/workarea/admin/create_segments/setup.html.haml

```diff
@@ -10,6 +10,8 @@
       = render_message 'error', message
 
     = form_tag create_segments_path, method: 'post' do
+      = hidden_field_tag :id, @segment.id unless @segment.new_record?
```

Comment:
We can also stuff this in the URL, but this seems like the way we're doing it everywhere else, so I figured why rock the boat.